### ISSUE TITLE
bug: Typing issue with Spotify for Podcasters

### ISSUE THREAD

**Issue by zcraber** [keymanapp/keyman#8594]
### Describe the bug

Unable to type `Episode Description` in Malayalam on [Spotify for Podcasters.](https://podcasters.spotify.com/)

### Reproduce the bug

_**Note**: This will require an account on [Spotify for Podcasters](https://podcasters.spotify.com/). It's a free podcasting service._
1. After logging in, if you already have an audio file uploaded, go to [Episodes](https://podcasters.spotify.com/pod/dashboard/episodes) and click on it. Else click on `New Episode` and upload audio.
2. Click on the `Edit` icon and scroll down to `Episode description`.
3. Type <kbd>i</kbd><kbd>m</kbd><kbd>g</kbd><kbd>l</kbd><kbd>e</kbd><kbd>e</kbd><kbd>s</kbd><kbd>h</kbd>.
4. You'll get `ഷ്`.

![spotify podcasters keyman issue](https://user-images.githubusercontent.com/61133303/232210682-3c3918de-ab36-476c-a7f9-8e8d77c75200.gif)

### Expected behavior

`ഇംഗ്ലീഷ്`

### Related issues

Malayalam can be typed without issues by toggling the `HTML` switch under the `Episode description`.
And there is no issue in typing Malayalam elsewhere on the website.

### Keyman apps

- [ ] Keyman for Android
- [ ] Keyman for iPhone and iPad
- [ ] Keyman for Linux
- [ ] Keyman for macOS
- [X] Keyman for Windows
- [ ] Keyman Developer
- [ ] KeymanWeb
- [ ] Other - give details at bottom of form

### Keyman version

16.0.139

### Operating system

Windows 10 22H2

### Device

Windows Laptop

### Target application

https://podcasters.spotify.com/

### Browser

Brave Version 1.50.118 Chromium: 112.0.5615.87

### Keyboard name

mozhi_malayalam

### Keyboard version

3.2.3

### Language name

Malayalam

**Comment by zcraber** [comment]
> Thank you for the feedback. We have scheduled time to investigate this in a few weeks. In the meantime, if you can discover which rich text component is in use on that page, that may help us to track the source of the problem down. Our immediate guess is that the component deletes clusters of Malayalam letters when backspace is pressed, rather than just deleting the right-most character.

Thanks for the info. They're using [Slate.js](https://github.com/ianstormtaylor/slate). I checked typing using the [demo](https://www.slatejs.org/examples/richtext) on their website and got the same result. So, I guess this is a bug with Slate.js?

**Comment by mcdurdin** [comment]
Yes, [our recent compatibility test with Slate.js](https://github.com/keymanapp/keyman/issues/7938#issuecomment-1361452798) shows that this is an issue. We have not yet taken this through to a bug report for Slate.js yet.

**Comment by zcraber** [comment]
The same issue occurs on Discord too. They use Slate.js as well.

**Comment by rc-swag** [comment]
@zcraber I was writing up a bug report for this issue on the Slate.js github page. I found that the demo link was still failing. However the sandbox link, https://codesandbox.io/p/sandbox/slate-reproductions-c7gyg  using the example of <kbd>i</kbd><kbd>m</kbd><kbd>g</kbd><kbd>l</kbd><kbd>e</kbd><kbd>e</kbd><kbd>s</kbd><kbd>h</kbd>.  it works as expected.
I wonder if there is a difference in versions used between the two links. Can you verify if this is still and issue with Spotify and Discord? If it is I will still raise the issue on slate.js

**Comment by AdelinaM17n** [comment]
>  Can you verify if this is still and issue with Spotify and Discord? If it is I will still raise the issue on slate.js
I've ran into this issue attempting to use Tamil Anjal Paangu keyboard with windows, So it appears to still at least be an issue with discord

**Comment by zcraber** [comment]
> Can you verify if this is still and issue with Spotify and Discord? If it is I will still raise the issue on slate.js

Sorry for the late reply. Seems like the issue is fixed in Slate.js as I couldn't replicate it in their demo. 
Unfortunately, the issue is still there in Spotify for Creators and Discord.
